docs/concepts/data-partitions.md

```diff
@@ -0,0 +1,95 @@
+# Overview of Data Partitions
+
+Data partitioning is an optional feature that can be enabled for a DICOM service. It implements a light-weight data partition scheme that enables customers to store multiple copies of the same image with the same identifying instance UIDs on a single DICOM service. 
+
+While UIDs **should** be [unique across all contexts](http://dicom.nema.org/dicom/2013/output/chtml/part05/chapter_9.html), it's common practice for DICOM files to be written to portable storage media by a healthcare provider and given to a patient, who then gives the files to another healthcare provider, who then transfers the files into a new DICOM storage system. Thus, multiple copies of one DICOM file commonly exist in isolated DICOM systems. As DICOM functionality moves to the cloud, unifying previously disconnected systems, data partitioning can provide an on-ramp for your existing data stores and workflows.
+
+## Feature Enablement
+The data partitions feature can be enabled by setting the configuration key `DicomServer:Features:EnableDataPartitions` to `true` through your local [appsettings.json](../../src/Microsoft.Health.Dicom.Web/appsettings.json) file or host-specific options.
+
+Once enabled, this feature adds a new data partition resource, creates an initial partition named `Microsoft.Default`, and modifies the API surface of the DICOM server. 
```

Comment:
The default partition record is created even before enabling the flag. Should we mention, if they try to access existing data they should include default partition in the URL

Reply:
updated